src/sections/services.html

```diff
@@ -0,0 +1,33 @@
+<div
+  class="flex flex-col items-center gap-24 lg:gap-0 lg:flex-row justify-around mt-12 px-8 sm:px-28 lg:px-52"
+>
+  <ul
+    class="lg:text-5xl xl:text-6xl text-black font-[500] leading-[100%] lg:w-large lg:h-large flex flex-col lg:gap-24 text-3xl gap-12"
+  >
+    <li>Brand strategy</li>
+    <li>Corporate Identity & Website Development</li>
+    <li>SEO & copyWriting</li>
+    <li>Media Relations</li>
+    <li>Digital Marketing & content production</li>
+    <li>Influencer relations</li>
+  </ul>
+  <div class="ml-auto">
+    <div class="relative w-max">
+      <img
+        src="/src/assets/Vector 5.svg"
+        alt="green-background-image"
+        class="relative w-[350px] md:w-full"
+      />
+      <p
+        class="absolute top-1/2 w-80 left-1/2 -translate-x-1/2 -translate-y-1/2 text-[#EFECEA] text-2xl font-[400]"
```

Comment:
Use the custom tailwind color you defined instead of the arbitrary one. It is the same color for the text as the bg. And also use predefined classes for the font weight.
```suggestion
        class="absolute top-1/2 w-80 left-1/2 -translate-x-1/2 -translate-y-1/2 text-primarybg text-2xl font-normal"
```